cmd/openshift-install/agent_internal_integration_test.go

```diff
@@ -267,9 +280,34 @@ func readIgnition(ts *testscript.TestScript, ignPath string) (config igntypes.Co
 	return config, err
 }
 
+// [!] expandFile `file...` can be used to substitute environment variables
+// references for each file specified
```

Comment:
```suggestion
// references for each file specified.
```
linter is complaining about this